hypershift-operator/controllers/hostedcluster/hostedcluster_controller.go

```diff
@@ -2475,6 +2475,35 @@ func reconcileAutoScalerDeployment(deployment *appsv1.Deployment, hc *hyperv1.Ho
 						},
 						Command: []string{"/usr/bin/cluster-autoscaler"},
 						Args:    args,
+						LivenessProbe: &corev1.Probe{
+							ProbeHandler: corev1.ProbeHandler{
+								HTTPGet: &corev1.HTTPGetAction{
+									Path:   "/metrics",
```

Comment:
why is this querying /metrics? shouldn't it be [/health-check](https://github.com/kubernetes/autoscaler/blob/222f71ea73dcd3bf1d331b9bbddfa060137249ba/cluster-autoscaler/main.go#L406)?

Reply:
When the source code of cluster-autoscaler is changed to add the health check endpoint, hypershift can use it, until then /metrics is the only available option.

Reply:
Not sure I follow, Isn't the link above showing exactly that? We are particularly using the image from the [ocp 4.10 version.](https://github.com/openshift/kubernetes-autoscaler/blob/release-4.10/cluster-autoscaler/main.go#L405)

Reply:
> Not sure I follow, Isn't the link above showing exactly that? We are particularly using the image from the [ocp 4.10 version.](https://github.com/openshift/kubernetes-autoscaler/blob/release-4.10/cluster-autoscaler/main.go#L405)

Endpoint is corrected, thanks.